doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -1,9 +1,9 @@
 [id='clustering-smart-router-install-proc_{context}']
 = Clustering {KIE_SERVERS} with Smart Router
 
-You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVER}s and aggregate data from different {KIE_SERVER}s. Smart Router uses aliases to perform as a proxy.
+You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVER} instances and aggregate data from different {KIE_SERVER} instances. Smart Router uses aliases to perform as a proxy.
 
-This section describes how to install Smart Router and configure it for a Red Hat {PRODUCT_SHORT} runtime environment.
+// This section describes how to install Smart Router and configure it for a Red Hat {PRODUCT_SHORT} runtime environment.
```

Comment:
```suggestion
// This section describes how to install Smart Router and configure it for a Red Hat {PRODUCT_SHORT} runtime environment.
```
Can this line just be deleted?

Reply:
Okay, I will delete it. For now it was commented out simply because it's your document :) but with your approval I'm deleting.